docs/metrics/v3.md

```diff
@@ -1,397 +1,409 @@
 # Metrics Version 3
 
-In metrics version 3, all metrics are available under the endpoint:
+In metrics version 3, all metrics are available under the following base endpoint:
 
 ```
 /minio/metrics/v3
 ```
 
-however, a specific path under this is required.
+To query metrics of a specific type, append the appropriate path to the base endpoint.
+You **must** append a path, as querying the base endpoint alone does not return metrics.
 
 Metrics are organized into groups at paths **relative** to the top-level endpoint above.
 
+Metrics are also available using the [MinIO Admin Client](https://min.io/docs/minio/linux/reference/minio-mc-admin.html) and the `mc admin prometheus metrics` command. For more information, see [Metrics and Alerts](https://min.io/docs/minio/linux/operations/monitoring/metrics-and-alerts.html) in the MinIO Documentation.
+
 ## Metrics Request Handling
 
-Each endpoint below can be queried at different intervals as needed via a scrape configuration in Prometheus or a compatible metrics collection tool.
+Each endpoint can be queried as needed via a scrape configuration in Prometheus or a compatible metrics collection tool. You should schedule scrape operations so a prior scrape completes before the next one begins.
 
-For ease of configuration, each (non-empty) parent of the path serves all metric endpoints that are at descendant paths. For example, to query all system metrics one needs to only scrape `/minio/metrics/v3/system/`.
+For ease of configuration, each (non-empty) parent of the path serves all the metric endpoints at its child paths. For example, to query all system metrics scrape `/minio/metrics/v3/system/`.
 
-Some metrics are bucket specific. These will have a `/bucket` component in their path. As the number of buckets can be large, the metrics scrape operation needs to be provided with a specific list of buckets via the `bucket` query parameter. Only metrics for the given buckets will be returned (with the bucket label set). For example to query API metrics for buckets `test1` and `test2`, make a scrape request to `/minio/metrics/v3/api/bucket?buckets=test1,test2`. 
+Some metrics are bucket specific. These will have a `/bucket` component in their path. As the number of buckets can be large, the metrics scrape operation must provide a list of buckets with the `bucket` query parameter. The endpoint then returns only metrics for the given buckets, with the bucket name in a `bucket` label.
```

Comment:
```suggestion
Metrics with a `/bucket` component in the path return results for each specified bucket in the deployment. These endpoints **require** providing a list of buckets as a `bucket` query parameter.  The endpoint then returns only metrics for the given buckets, with the bucket name in a `bucket` label.
```

Reply:
This seems important. Maybe make it an H3 and reference it for metric groups that have bucket components.

Reply:
Reworked this and following paragraph so using H3 makes more sense